klippy/configfile.py

```diff
@@ -223,7 +223,7 @@ def _resolve_include(self, source_filename, include_spec, fileconfig,
         dirname = os.path.dirname(source_filename)
         include_spec = include_spec.strip()
         include_glob = os.path.join(dirname, include_spec)
-        include_filenames = glob.glob(include_glob)
+        include_filenames = glob.glob(include_glob, recursive=True)
```

Comment:
The `recursive` argument for `glob.glob()` was added in Python 3.5.  AFAIK Klipper still supports running on Python 2.7, so this would be a breaking change.